### PR TITLE
feat: implement profile page

### PR DESCRIPTION
**Pull request by milo219**

### CODE REVIEW

**Comment by demons1502** [comment]
Conflict

File: src/pages/Profile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+            </div>
+            <div className=" d-flex col-md-12 justify-content-end">
+              <Button classButton="btn-ChangePhoto" nameButton="Changes" />
+            </div>
+            <div className="block my-5">
+              <h5 className="title fw-bold mb-5">Connected Account</h5>
+              <div className="cube mb-4">
+                <div className="list-item row mb-4">
+                  <span className="col-md-6 ">
+                    <svg
```
Comment (demons1502) [inline src/pages/Profile/index.tsx:65]: Move to `assets`

File: src/pages/Profile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                  </div>
+                  <FormInput
+                    classForm="form-input"
+                    placeholder="Nguyễn Mai Lan"
+                  />
+                </div>
+              </div>
+              <div className="list-item row ">
+                <span className="col-md-6">
+                  <svg
```
Comment (demons1502) [inline src/pages/Profile/index.tsx:96]: Same

File: src/pages/Profile/index.tsx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+              </div>
+              <div className="col-md-8 mb-3 d-flex align-items-center">
+                <Button
+                  classButton="btn-ChangePhoto btn-success btn-block"
+                  nameButton="Delete"
+                />
+              </div>
+            </div>
+            <div className="row mt-5">
+              {dataMock.dataFormInputProfile.map((listForm) => (
```
Comment (demons1502) [inline src/pages/Profile/index.tsx:50]: add `form` and `TODO` to handle submit

File: src/pages/Profile/index.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                  </span>
+                  <div className=" d-flex col-md-6 justify-content-end">
+                    <Button
+                      classButton="btn-Disconnect"
+                      nameButton="Disconnect"
+                    />
+                  </div>
+                  <FormInput
+                    classForm="form-input"
+                    placeholder="Nguyễn Mai Lan"
```
Comment (baonguyen26) [inline src/pages/Profile/index.tsx:78]: ???

File: src/pages/Profile/index.tsx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+                <div className=" d-flex col-md-6 justify-content-end">
+                  <Button
+                    classButton="btn-Disconnect"
+                    nameButton="Disconnect"
+                  />
+                </div>
+                <FormInput
+                  type="email"
+                  classForm="form-input"
+                  placeholder="mailan@gmail.com"
```
Comment (baonguyen26) [inline src/pages/Profile/index.tsx:96]: example@gmail.com

**Review by demons1502 — APPROVED** [review]

**Pull request merged**